unstructured/ingest/runner/writers/elasticsearch.py

```diff
@@ -0,0 +1,24 @@
+import typing as t
+from dataclasses import dataclass
+
+from unstructured.ingest.interfaces import BaseDestinationConnector
+from unstructured.ingest.runner.writers.base_writer import Writer
+
+if t.TYPE_CHECKING:
+    from unstructured.ingest.connector.elasticsearch import (
+        ElasticsearchWriteConfig,
+        SimpleElasticsearchConfig,
+    )
+
+
+@dataclass
+class ElasticsearchWriter(Writer):
+    connector_config: "SimpleElasticsearchConfig"
+    write_config: "ElasticsearchWriteConfig"
+
+    def get_connector_cls(self, **kwargs) -> BaseDestinationConnector:
```

Comment:
i don't think the kwargs are needed anymore?

Reply:
[remove kwargs, inherit fields, add num-processes for upload to docs](https://github.com/Unstructured-IO/unstructured/pull/2152/commits/04fe747b437792ae68705a6ceed9970f9d6c221b)